### ISSUE TITLE
Segmentation fault in MDE module

### ISSUE THREAD

**Issue by 0mar** [0mar/mercurial#1]
When enabling minimal distance enforcement, the simulation often crashes.
Reproducable with 

``` bash
py main.py -df configs/dynamic.ini -o scenes/doolhof.json -n 2000
```

The problem probably lies in `mde.f90`.

**Comment by 0mar** [comment]
I think this is caused by the assumption that the positions fed into `mde.f90` fall within the scope of the scene.

**Comment by 0mar** [comment]
Fixed in b668857
